core/overview.go

```diff
@@ -6,24 +6,28 @@ import (
 	"github.com/textileio/textile-go/repo"
 )
 
-// Overview is a wallet overview object
-type Overview struct {
+// Summary is a wallet summary object
+type Summary struct {
 	AccountPeerCount int `json:"account_peer_cnt"`
 	ThreadCount      int `json:"thread_cnt"`
 	FileCount        int `json:"file_cnt"`
 	ContactCount     int `json:"contact_cnt"`
 }
 
-// Overview returns an overview object
-func (t *Textile) Overview() (*Overview, error) {
+// Summary returns a summary of node data
+func (t *Textile) Summary() (*Summary, error) {
```

Comment:
Is this exposed via the REST API? If not, it defs should be. Would be super useful to have.

Reply:
oh yeah, should be for sure